Includes/networkManager.cpp

```diff
@@ -24,10 +24,12 @@ void NetworkManager::thread_main(NetworkManager *manager) {
 #ifdef NXDK
   const Config &config = manager->config;
```

Comment:
nit: Consider changing this to `const netConfig& config = &manager->config.settings.net;`, as no other parts of the `Config` class are used.

Reply:
Good point, changed.